docs/excel/custom-functions-best-practices.md

```diff
@@ -26,7 +26,14 @@ As described in the [custom functions overview](custom-functions-overview.md) ar
 The following code sample shows how to do this association. The sample defines the custom function `add` and associates it with the object in the JSON metadata file where the value of the `id` property is **ADD**.
 
 ```js
-function add(first, second){
+/**
+ * Add two numbers
+ * @customfunction 
+ * @param {number} first First number
+ * @param {number} second Second number
```

Comment:
Consider adding a hyphen between the param names and the description. Also, add periods to the ends of the descriptions.

Reply:
Weirdly with the way that this thing works the hyphen then would show in Intellisense so I'm gonna do the periods but not the hyphens